.github/workflows/coverity-scan.yml

```diff
@@ -0,0 +1,88 @@
+name: Coverity Scan
+on:
+  schedule:
+    - cron: '0 0 1 * *' # Monthly (1st day of month at midnight)
+
+  workflow_dispatch: # Mainly for testing. Don't forget the Coverity usage limits.
```

Comment:
This is new to me, care to elaborate?

Reply:
It allows to manually run the workflow. It adds a button in the `Actions` page of the repo for this workflow that triggers the workflow. See: https://docs.github.com/en/actions/managing-workflow-runs/manually-running-a-workflow